src/software/ai/navigator/obstacle/obstacle_factory.h

```diff
@@ -52,8 +67,70 @@ class ObstacleFactory
      *
      * @return A list of obstacles representing the given team
      */
-    std::vector<Obstacle> getVelocityObstaclesFromTeam(const Team &team);
+    std::vector<Obstacle> createVelocityObstaclesFromTeam(const Team &team);
+
+    /**
+     * Create circle obstacle around robot with additional radius scaling
+     *
+     * @param robot_position robot position
+     * @param radius_scaling how much to scale the radius
+     *
+     * @return obstacle around the robot
+     */
+    Obstacle createRobotObstacle(const Point &robot_position,
+                                 const double radius_scaling = 1.0);
```

Comment:
Why do we have `radius_scaling` but also `shape_expansion_amount`?

Reply:
originally it's to support VelocityObstaclePathManagerConfig, but that's redundant, so I'm getting rid of radius_scaling and VelocityObstaclePathManagerConfig